src/ripple/app/paths/PathRequest.cpp

```diff
@@ -480,8 +480,9 @@ Json::Value PathRequest::doUpdate (RippleLineCache::ref cache, bool fast)
 
         if (valid)
         {
-            MetaView sandbox(
-                cache->getLedger(), tapNONE);
+            boost::optional<PaymentView> sandbox;
+            sandbox.emplace(cache->getLedger(), tapNONE);
```

Comment:
I have no problem with your clever use of `optional` in this block, but when you use a type in way it isn't normally used a brief comment is nice.